nautobot/docs/models/dcim/cable.md

```diff
@@ -32,3 +32,12 @@ Traced from Interface 1 on Device A, Nautobot will show the following path:
 * Cable 1: Interface 1 to Front Port 1
 * Cable 2: Rear Port 1 to Rear Port 2
 * Cable 3: Front Port 2 to Interface 2
+
+A cable can also be traced through a circuit.  Due to the path caching, the Circuit must have both terminations added before connecting a cable.
```

Comment:
Is the path caching still an issue with your latest diffs? If so, any ideas as to what would be needed to remove this limitation?

Reply:
Yes, it is still a limitation.  This is due to the the Termination being a PathEndpoint.  If only one termination exists, then the CablePath assumes this is desired and sets the Destination as the CircuitTermination. This can be seen in test test_105_interface_to_circuittermination.  If you add the second termination before the first cable, the Destination will be changed from CircuitTermination to None, which allows the path refresh to pick up on the second termination and complete the path once the second cable is added.

Reply:
One solution is to change the status of the Cable connected to the first termination, if you decide to later add a second termination. I will update the docs accordingly.